File: src/main/java/com/fendyk/DTOs/ActivitiesDTO.java
package com.fendyk.DTOs;

import java.util.ArrayList;

public class ActivitiesDTO {

    public String id;
    public String minecraftUserId;

    public ActivityDTO time;
    public ArrayList<ActivityDTO> mining;
    public ArrayList<ActivityDTO> pvp;
    public ArrayList<ActivityDTO> pve;

    public ActivityDTO getTime() {
        return time;
    }

    public String getId() {
        return id;
    }

    public String getMinecraftUserId() {
        return minecraftUserId;
    }

    public ArrayList<ActivityDTO> getMining() {
        return mining;
    }

    public ArrayList<ActivityDTO> getPvp() {
        return pvp;
    }

    public ArrayList<ActivityDTO> getPve() {
        return pve;
    }
}

File: src/main/java/com/fendyk/DTOs/ActivitiesDTO.kt
package com.fendyk.DTOs

data class ActivitiesDTO(val id: String) {
    var minecraftUserId: String? = null
    var time: ActivityDTO? = null
    var mining: ArrayList<ActivityDTO> = ArrayList()
    var pvp: ArrayList<ActivityDTO> = ArrayList()
    var pve: ArrayList<ActivityDTO> = ArrayList()
}
